src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1458,7 +1458,7 @@ internal static Encoding GetEncoding(string path)
                 return Encoding.Unicode;
             }
 
-            return Encoding.ASCII;
+            return new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Comment:
I guess you can use the static `utf8NoBom` declared below:
```
internal static readonly UTF8Encoding utf8NoBom =
            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Reply:
@Gimly please apply this change

Reply:
Done, you're right I hadn't seen the constant.